packages/create-llama/helpers/types.ts

```diff
@@ -15,8 +27,8 @@ export interface InstallTemplateArgs {
   template: TemplateType;
   framework: TemplateFramework;
   engine: TemplateEngine;
-  contextFile?: string;
   ui: TemplateUI;
+  dataSource: TemplateDataSource;
```

Comment:
@leehuwuj did you test this with llamapacks and community templates? pretty sure dataSource would then be undefined and your code might crash

Reply:
@marcusschiesser they are working well, but you are right, we may have undefined dataSource so let me change it to optional attribute